tests/scripts/github-action-helper.sh

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env bash
+
+set -xe
+
+function install_deps() {
+    sudo wget https://github.com/mikefarah/yq/releases/download/3.4.1/yq_linux_amd64 -O /usr/bin/yq
+    sudo chmod +x /usr/bin/yq
+}
+
+function print_k8s_cluster_status() {
+    kubectl cluster-info
+    kubectl get pods -n kube-system
+}
+
+function use_local_disk() {
+    BLOCK=$(sudo lsblk --paths | awk '/14G/ {print $1}' | head -1)
```

Comment:
Instead of calling this repeatedly in each function just make it global at the beginning of the script.

Reply:
done.